www/api/class/centreon_metric.class.php

```diff
@@ -155,6 +155,44 @@ protected function getListByService()
         );
     }
 
+    /**
+     * Get last metrics value
+     *
+     * @return array
+     */
+    public function getLastMetricsData()
+    {
+        if (false === isset($this->arguments['services']) &&
```

Comment:
Shouldnt compare boolean with boolean to get another boolean, in a boolean evaluated expression :

if (!isset($this->arguments['services']) && !isset($this->arguments['metrics'])

Reply:
i have followed code from centreon php team. Me, i prefer your syntax.